app/src/main/java/fi/helsinki/opintoni/domain/portfolio/PortfolioComponent.java

```diff
@@ -27,5 +27,6 @@
     FAVORITES,
     CREDITS,
     FREE_TEXT_CONTENT,
-    LANGUAGE_PROFICIENCIES
+    LANGUAGE_PROFICIENCIES,
+    SKILLS_AND_EXPERTISE
```

Comment:
Tämähän ei ole varsinaisesti uusi portfolion komponentti (ts. komponenttityyppi mitä tämä Enum mallintaa, vaan `FREE_TEXT_CONTENT`in instanssi). Olisi siistimpi palauttaa clientille järjestys siten että tästä Enumista muodostettuun listaukseen katenoidaan jonkun `PreconfiguredStudentFreeTextContentItems` etc. Enumin järjestetty listaus.

Ja jos/kun tuolla tiedolla ei tehdä bäkkärissä mitään, niin olisi parempi siirtää tuo `FREE_TEXT_CONTENT`-itemien lista clienttiin (missä se katenoidaan bäkkärin palauttamaan komponenttityyppilistaan).